internal/handlers/telegram/handler.go

```diff
@@ -56,7 +57,7 @@ func stickerHandler(tg *Client, u tgbotapi.Update) {
 	formatted := tg.Settings.Prefix + u.Message.From.UserName +
 		tg.Settings.Suffix + u.Message.Sticker.Emoji
 
-	tg.sendToIrc(formatted)
+	tg.sendToIrc(strings.Trim(formatted," "))
```

Comment:
What is this trimmed for?

Reply:
There was an extra space at the end of the string. It was not expected, so I'm trimming it

Reply:
@kennedy This would probably be a good place to leave a comment for context. We'll probably forget why we did this in another twelve months, maybe. :smile: